### PR TITLE
RavenDB-16822  OLAP ETL errors should be shown

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts [hunk 1]
```diff
@@ -1851,6 +1851,15 @@ class ongoingTasksStats extends viewModelBase {
                     case "Sql":
                     case "Olap": {
                         const elementWithData = context.rootStats as EtlPerformanceBaseWithCache;
+                        
+                        if (elementWithData.HasTransformErrors) {
+                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Transform errors:<div class="value">Open notification center for details </div></div>`;
```
Comment (arekpalinski) [inline src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts]:
```suggestion
                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Transform errors:<div class="value">Open Notification center for details </div></div>`;
```

Comment (arekpalinski) [reply src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts]: For naming consistency:

image

File: src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts [hunk 2]
```diff
@@ -1851,6 +1851,15 @@ class ongoingTasksStats extends viewModelBase {
...
+                        }
+                        
+                        if (elementWithData.HasLoadErrors) {
+                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Load errors:<div class="value">Open notification center for details </div></div>`;
```
Comment (arekpalinski) [inline src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts]:
```suggestion
                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Load errors:<div class="value">Open Notification center for details </div></div>`;
```

File: src/Raven.Studio/wwwroot/App/views/database/tasks/editOlapEtlTask.html [hunk 3]
```diff
@@ -54,7 +54,7 @@ <h3 data-bind="text: $parent.isAddingNewOlapEtlTask() ? 'New OLAP ETL' : 'Edit O
                             </div>
                             <div data-bind="validationElement: customPartition, collapse: customPartitionEnabled">
                                 <div class="form-group" data-bind="validationElement: customPartition">
-                                    <label for="taskName" class="control-label">Custom Partition</label>
+                                    <label for="taskName" class="control-label">Partition Value</label>
```
Comment (arekpalinski) [inline src/Raven.Studio/wwwroot/App/views/database/tasks/editOlapEtlTask.html:57]: Why this got changed as well? Is that correct? @aviv86?

File: src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts]: done, unified all places to `Notification Center`

File: src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts]: done, unified all places to `Notification Center`

File: src/Raven.Studio/wwwroot/App/views/database/tasks/editOlapEtlTask.html (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/tasks/editOlapEtlTask.html:57]: @aviv86 
Imho, writing `Partition Value` is more accurate than `Custom Partition`
It is the **value** that is entered there, not the partition name

**Review by ml054 — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**